torch/onnx/symbolic_opset10.py

```diff
@@ -137,8 +137,11 @@ def __interpolate(g, input, size, scale_factor, mode , align_corners, recompute_
 
 def _slice(g, input, axes, starts, ends, steps=None, dynamic_slice=False):
     if dynamic_slice:
+        print("start, end", starts, ends)
         starts = g.op("Unsqueeze", starts, axes_i=[0])
         ends = g.op("Unsqueeze", ends, axes_i=[0])
+        if isinstance(axes, int):
```

Comment:
What if dynamic_slice = False and axes is an integer?
Would it be a case we need to check?

Reply:
when dynamic_slice is False axes should be an integer. This check is only needed when dynamic_slice is True, because Unsqueeze operation cannot be performed on an int.